service/controller/resource/pvc/create.go

```diff
@@ -25,8 +25,9 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		r.logger.Debugf(ctx, "creating the PVCs in the Kubernetes API")
 
 		namespace := key.ClusterNamespace(customObject)
-		for _, PVC := range pvcsToCreate {
-			_, err := r.k8sClient.CoreV1().PersistentVolumeClaims(namespace).Create(ctx, PVC, v1.CreateOptions{})
+		for _, persistentVolumeClaim := range pvcsToCreate {
+			toCreate := persistentVolumeClaim
```

Comment:
Are you doing this just for the rename? You also might need a deep copy here if you don't want to lose half of the values (arrays for example are not going to be copied afaik).

Reply:
This is mainly to work around a spurious (IMO) warning in `golangci-lint`. Range for loops reuse the same variable on each iteration, so it's usually an error to take the address of the loop variable as in the line below. In this case I don't believe it matters because `r.k8sClient.CoreV1().PersistentVolumeClaims(namespace).Create` doesn't change the object that is passed in. I could have also added `//nolint`. Maybe that would be clearer?